internal/test/smoke/domain_update_system_test.go

```diff
@@ -108,7 +124,7 @@ func (s *SuiteDomainUpdateAgent) TestUpdateDomain() {
 			Expected: TestCaseExpect{
 				StatusCode: http.StatusBadRequest,
 				BodyFunc: WrapBodyFuncErrorResponse(func(t *testing.T, body *public.ErrorResponse) error {
-					assert.Equal(t, builder_api.NewErrorResponse().
+					require.Equal(t, builder_api.NewErrorResponse().
```

Comment:
This helps me to fix one by one, instead of list all the issues mixed with all the logs.

Reply:
`-test.failfast ` But this option is already used in smoke tests. So it also depends how the tests are written, e.g. this `require` vs `assert` Where `require` is an `assert` with additonal `t.FailNow()`.